src/index.html

```diff
@@ -2,11 +2,412 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">    <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="href=#home">
+        <img
+          class="logo"
+          src="\images\logo.png"
+          alt="moyoLogo"
+        >
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a class="nav__link is-active" href="#">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">smartphones</a>
+          </li>
+
+          <li class="nav__item">
+            <a
+              class="nav__link"
+              href="#"
+              data-qa="nav-hover"
+            >
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page">
+      <div class="page__card">
+        <article class="card" data-qa="card">
+          <img
+              class="card__photo"
+              src="images/imac.jpeg"
+              alt="imac"
+            >
+
+          <p class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</p>
+
+          <p class="card__text">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="card__stars">
+              <div class="card__star"></div>
+
+              <div class="card__star"></div>
+
+              <div class="card__star"></div>
+
+              <div class="card__star"></div>
+
+              <div class="card__star"></div>
```

Comment:
Don't add empty lines between single-line siblings.
```suggestion
              <div class="card__star"></div>
              <div class="card__star"></div>
              <div class="card__star"></div>
              <div class="card__star"></div>
              <div class="card__star"></div>
```